### ISSUE TITLE
Links within m.notice are unclickable

### ISSUE THREAD

**Issue by BrainBlasted** [vector-im/riot-android#2281]
As the title says, it is impossible to click on links within m.notices on Riot-android. Here is a sample message:

```
{
  "content": {
    "body": "[World/fractal] Daniel Garcia Moreno merged merge request a11y: add missing accessible names to some buttons (#116)",
    "msgtype": "m.notice",
    "formatted_body": "[World/fractal] Daniel Garcia Moreno merged merge request \u003ca href\u003d\u0027https://gitlab.gnome.org/World/fractal/merge_requests/116\u0027\u003ea11y: add missing accessible names to some buttons (#116)\u003c/a\u003e",
    "format": "org.matrix.custom.html"
  },
  "event_id": "$15269117395470UpHLH:t2bot.io",
  "m_is_internal_pagination_token": false,
  "m_sent_state": "SENT",
  "origin_server_ts": 1526911739288,
  "room_id": "!hwiGbsdSTZIwSRfybq:matrix.org",
  "sender": "@gitlab:t2bot.io",
  "type": "m.room.message",
  "unsigned": {
    "age": 32511
  }
}
```

**Comment by tulir** [comment]
Probably related to #2133 and #1037

**Comment by vsatmydynipnet** [comment]
Probably related to
https://github.com/vector-im/riot-android/issues/2687

It really would be great if this could be fixed. Running maubot with rss plugin and with NOTIZ links are not clickable on Android, as TEXT links works, but counter/notify is a huge problem.

thank you very much.

**Comment by bmarty** [comment]
@BillCarsonFr I think we can close this one ?

**Comment by KsmoinO** [comment]
Hello

Since last release (https://github.com/vector-im/riot-android/releases/tag/v0.8.24) links in plain text, in m.notice message type, from bots (like RSS bot) are unclickable and does not appear as links too.

Thank you.

**Comment by bmarty** [comment]
Yes, it's fixed in 0.8.25

**Comment by KsmoinO** [comment]
I confirm it works. Thanks :)
